Tests/Simulation/IntegrationTestFixtureSim.cs

```diff
@@ -64,7 +64,7 @@ public override void SetupTestDevices()
             Device1003_Simulated_HttpBasedDecoder = new TestDeviceInfo
             {
                 DeviceID = "0000000000001003",
-                AppEUI = "0000000000001003",
+                AppEUI = GetJoinEui(1003),
```

Comment:
`0000000000001003` was in hex so shouldn't this read `GetJoinEui(0x1003)`?

Reply:
Resolving this as per other comment